repose-aggregator/functional-tests/spock-functional-test/src/test/configs/features/core/intrafilterlogging/header-normalization.cfg.xml

```diff
@@ -22,14 +22,6 @@
             </whitelist>
         </target>
 
-        <!-- applies to all POST and PUT requests -->
-        <target http-methods="POST PUT">
-            <whitelist id="modification">
-                <header id="X-THIRD-Filter"/>
-                <header id="x-auth-token"/>
-            </whitelist>
-        </target>
-
```

Comment:
Is it just me, or is it odd that the intra-filter logging tests use header-normalization to being with? Perhaps other filters as well. Not that it's bad to test the behavior of multiple filters working together, we should do that, but hopefully we have tests that minimize the scope (to only the filter under test) so that we can assert something about the behavior of the filter and not just one specific composition of filters.